src/Routes.js

```diff
@@ -1,23 +1,30 @@
+//#region > Imports
 //> React
 // Contains all the functionality necessary to define React components
-import React from 'react';
+import React from "react";
 // DOM bindings for React Router
-import { Route, Switch } from 'react-router-dom';
+import { Route, Switch } from "react-router-dom";
 
 //> Components
 /**
  * HomePage: A basic template page
  */
```

Comment:
This description is no longer needed, once it serves little purpose. It should be reworked with `@description` for each and every imported page.